examples/videocomposer/vc/engine/inference_single_lite.py

```diff
@@ -0,0 +1,225 @@
+            else:
+                depth_data = None
+
+            if "canny" in cfg.video_compositions and "canny" in cfg.guidances:
+                canny_data = canny_extractor(misc_data)
+            else:
+                canny_data = None
+
+            if "sketch" in cfg.video_compositions and ("sketch" in cfg.guidances or "single_sketch" in cfg.guidances):
+                if cfg.read_sketch:
+                    sketch_data = np.tile(frame_sketch[:, None, ...], (batch_size, frames_num, 1, 1, 1))
+                else:
+                    sketch_data = misc_data
+                sketch_data = wrapped_predict(sketch_extractor, sketch_data)
+            else:
+                sketch_data = None
+
+            if "single_sketch" in cfg.video_compositions and "single_sketch" in cfg.guidances:
+                single_sketch_data = np.tile(sketch_data[:, :, :1], (1, 1, frames_num, 1, 1))
+            else:
+                single_sketch_data = None
+
+            b, f, _, h, w = video_data.shape
+            # TODO: load from config
+            c = 4
+            h = h // 8
+            w = w // 8
+            if cfg.share_noise:
+                noise = np.random.randn(b, c, h, w).astype(np.float32)
+                noise = np.tile(np.expand_dims(noise, 2), (1, 1, f, 1, 1))
+            else:
+                noise = np.random.randn(b, c, f, h, w).astype(np.float32)
+
+            full_model_kwargs = {
+                "y": text_embs,
+                "local_image": image_local,
+                "image": img_embs,
+                "depth": depth_data,
+                "canny": canny_data,
+                "sketch": sketch_data,
+                "masked": masked_video,
+                "motion": mv_data_video,
+                "single_sketch": single_sketch_data,
+                "fps": fps,
+            }
+
+            # Save generated videos
+            # --------------------------------------
+            partial_keys = cfg.guidances
+            model_kwargs = prepare_lite_model_kwargs(
+                partial_keys=partial_keys,
+                full_model_kwargs=full_model_kwargs,
+                use_fps_condition=cfg.use_fps_condition,
+            )
+            diffusion_output = diffusion(
+                noise,
+                model_kwargs=model_kwargs,
+                guide_scale=9.0,
+                timesteps=cfg.ddim_timesteps,
+                eta=0.0,
+            )
+            video_output = wrapped_predict(decoder, diffusion_output)
+
+            end = time.time()
+            time_cost[step, trial] = end - start
+
+            visualize_with_model_kwargs(
+                model_kwargs=model_kwargs,
+                video_data=video_output,
+                ori_video=misc_backups,
+                caps=caps,
+                fname=f"batch_{step}_trial_{trial}.gif",
+                cfg=cfg,
+            )
+
+    _logger.info("Congratulations! The inference is completed!")
```

Comment:
add log for where the outputs are saved for easy lookup.

Reply:
done.